index.html

```diff
@@ -16,17 +16,20 @@
 
     <section class="desktop-header">
       <header>
-        <nav class="desktop-nav">
-          <a href="#section-1">Portfolio</a>
-          <a href="#section-2">About</a>
-          <a href="#section-3">Contact</a>
-        </nav>
-        <nav class="mobile-nav">
-          <a href=""><img src="images/Logo Placeholder.svg" alt="My logo" /></a>
-
-          <a href="#section-1"
-            ><img src="images/Icon - Menu.svg" alt="Hamburger menu"
+        <nav class="navbar">
+          <a href="#"
+            ><img class="" src="images/Logo Placeholder.svg" alt="My logo"
           /></a>
+
+          <ul class="nav-menu">
+            <li class="nav-item"><a href="#section-1">Portfolio</a></li>
+            <li class="nav-item"><a href="#section-2">About</a></li>
+            <li class="nav-item"><a href="#section-3">Contact</a></li>
+          </ul>
+
+          <div class="hamburger">
+            <img src="images/Icon - Menu.svg" alt="Hamburger menu" />
+          </div>
```

Comment:
- [ ] When the hamburger is clicked it looks different than on design, kindly handle this:
Yours:
![image](https://user-images.githubusercontent.com/62705947/208680137-4be817a8-fb2b-455d-ac48-071419c4ebd6.png)
Desing:
![image](https://user-images.githubusercontent.com/62705947/208680049-ce96f6d3-4254-4e37-a10c-bf9c16ea1194.png)

Reply:
- [ ] Welcome should  be present only on mobile
![image](https://user-images.githubusercontent.com/62705947/208680347-d75a4a37-6204-4e5e-a74e-f6233f471bba.png)

Reply:
- [ ] After I open the hamburger menu and click on any of the options, they send me to the correct part of the page, but after that, I cannot move on your app anymore, kindly check that

Reply:
- [ ] According to the design the welcome and hamburger button should be on the same line
![image](https://user-images.githubusercontent.com/62705947/208680812-8ea0eca0-1cc1-4bd3-bd0e-a3aac6f4bc84.png)